### ISSUE TITLE
Can't the code run yet?

### ISSUE THREAD

**Issue by Liuys-szu** [whduddhks/Anomaly-Detection-in-Video-via-Self-Supervised-and-Multi-Task-Learning#1]
Hi! I cloned your code and want to train but I found some modules are missing.

**Comment by whduddhks** [comment]
First, You have to git clone the yolo v3 model from here : https://github.com/ultralytics/yolov3
Second, add yolov3/detect_loc.py and change the file in yolov3/utils/general.py

Third, you have to download the dataset from here : https://github.com/StevenLiuWen/ano_pred_cvpr2018

And I will check I upload every files.
But When I trained it, it's results is not good :(

You can get the code from here : https://github.com/lilygeorgescu/AED-SSMTL

Thank you

**Comment by Liuys-szu** [comment]
> First, You have to git clone the yolo v3 model from here : https://github.com/ultralytics/yolov3 Second, add yolov3/detect_loc.py and change the file in yolov3/utils/general.py
> 
> Third, you have to download the dataset from here : https://github.com/StevenLiuWen/ano_pred_cvpr2018
> 
> And I will check I upload every files. But When I trained it, it's results is not good :(
> 
> You can get the code from here : https://github.com/lilygeorgescu/AED-SSMTL
> 
> Thank you

Thank you very much, I ran the code successfully, and waiting for the result. I had got the source code from the original author, but I am not familiar with Tensorflow and encountered many problems. 

I found your code and I want to learn more details from the code, and you helped me a lot, thank you!

**Comment by whduddhks** [comment]
Thank you.

And I'm waiting for original code from author.
So when i take it, the code will be updated.

Good Luck

**Comment by Liuys-szu** [comment]
Thank you! But I got an issue when I ran the inference.py.

"All frames were detected, begin to compute AUC.
Traceback (most recent call last):
  File "inference.py", line 141, in <module>
    inference(test_cfg)
  File "inference.py", line 129, in inference
    assert scores.shape == labels.shape, f'Ground truth has {labels.shape[0]} frames, but got {scores.shape[0]} detected frames.'
AssertionError: Ground truth has 1938 frames, but got 1932 detected frames."

I don't know why the labels.shape != scores.shape. Have you ever encountered this problem?
